### ISSUE TITLE
nonceTPM

### ISSUE THREAD

**Issue by ernesto418** [tpm2-software/tpm2-tools#2852]
Is there any tool that provides the current nonce (nonceTPM) of the authsession ?

I the past I used:

```
tpm2 startauthsession -S session.ctx --policy-session
tpm2_sessionconfig session.ctx
```
But from this commit a389c995873c19fb4d3662c3ba6a75d8a093645e, `tpm2_sessionconfig`  is not providing the information any more.

Regards

**Comment by idesai** [comment]
@erezgeva indeed, we dropped the support because in context of the tpm2-tools, modifying the session attributes was the only useful bit of control we needed over the session data when passed between the different tools.
Did you have a particular use for it?

**Comment by idesai** [comment]
You just have to enable it with -x and it will be factored in.

```bash
tpm2_startauthsession -S session.ctx --policy-session
tpm2 policysigned -S session.ctx -c signing_key.ctx --raw-data with_nonce.bin -x
tpm2_flushcontext session.ctx
```
In the example above, you can now sign with_nonce.bin.
